dist/index.js

```diff
@@ -4636,9 +4636,14 @@ var requiredArgOptions = {
   trimWhitespace: true
 };
 var pagerdutyApiKey = core.getInput('pagerduty-api-key', requiredArgOptions);
-var serviceIdInput = core.getInput('service-id', requiredArgOptions);
+var serviceIdInput = core.getInput('service-id', { trimWhitespace: true });
+var serviceIdsInput = core.getInput('service-ids', { trimWhitespace: true });
```

Comment:
These both might need default value set in action.yml.  Otherwise when you go to trim() they might throw an error here.  Also when you go to concatenate if the id is empty you'll end up with an "" for the service id in the first object.  If the ids is empty it will throw an error when it tries to do a split on it.

So they might need default values and/or normalization here.

Reply:
There is a `filter` that is being performed prior to performing a map on the list that prevents empty "" or null ids.  So it should not impact my `trim()`.

When it comes to the `split` you are right it will error on a `null`.

I can either:
# Set the default inputs to an empty string that should should prevent any issues with the split on a `null`.
# Or, add a ternary:
```
.concat(serviceIdsInput ? serviceIdsInput.split(',') : [])
```

Reply:
I don't think I have a preference for dealing with the serviceIds.  

I never remember how the getInput options are going to work though, will they be happy with the {trimWhitespace: true} for a null object?  Maybe the default values on the action inputs are a better way to go just to avoid any issues with that?

Reply:
The documentation I have read states that a trim is performed on the inputs by default.  The `trimShitespace` was introduced to disable that.
https://github.com/actions/toolkit/issues/339

So I would assume on an input that is not provided, it doesn't fail.  But I can test it out and report my findings here.

Reply:
Test where the values are not provided.  There is no error on trim:
![image](https://user-images.githubusercontent.com/8923247/154729445-e24327c8-0368-4b52-8440-c9100b65552e.png)

Reply:
Test where inputs are set to empty string.  The correct error is shown.  GitHub treats it just like of the input was not provided at all.

![image](https://user-images.githubusercontent.com/8923247/154732025-f3c664a6-70cf-41c0-9ece-cb5a55bf12fd.png)

Reply:
Its my conclusion that we don't need to worry about `trimWhitespace ` and can actually remove that attribute.